lib/qless/server/views/_job.erb

```diff
@@ -43,9 +43,14 @@
 	            </ul>
 	          </div>
             <div class="btn-group custora-job-button-group">
-              <% if (pt_tag = (job.tags || []).find { |tag| tag.match(/pt-(.*)/) }).nil? %>
-                <button title="Create Pivotal Story" class="btn btn-info" onclick="confirmation(this, 'Create Pivotal Story?', function() { createPivotalStory('<%= job.jid %>', fade) })"><i class="glyphicon glyphicon-copy"></i></button>
-              <% else %>
+              <% if (pt_tags = (job.tags || []).select { |tag| tag.match?(/pt-(.*)/) }).none? %>
+                <button title="Create JIRA Issue" class="btn btn-info" onclick="confirmation(this, 'Create JIRA Issue?', function() { createJIRAIssue('<%= job.jid %>', fade) })"><i class="glyphicon glyphicon-copy"></i></button>
+                <button title="Create Pivotal Story" class="btn btn-default" onclick="confirmation(this, 'Create Pivotal Story?', function() { createPivotalStory('<%= job.jid %>', fade) })"><i class="glyphicon glyphicon-copy"></i></button>
+							<% end %>
+              <% if pt_tags.find { |tag| tag.match(/pt-jira-(.*)/) } %>
+							  <button title="View JIRA Issue" class="btn btn-info" onclick="window.open(window.jira.url + '/browse/<%= $1 %>', '_blank')"><i class="glyphicon glyphicon-new-window"></i></button>
+							<% end %>
+							<% if pt_tags.find { |tag| !tag.match?(/jira/) && tag.match(/pt-(.*)/) } %>
```

Comment:
Yeah, the indentation here is a mess 👎🏻 Trying to convert the entire file to either tabs or spaces changes almost every line. Not sure it's worth going through it at this point to correct it, but either way I think we should hold off for a different PR.

Opted for `match` for the second check because `match?` won't capture the matches into `$1`, which I'm using on the next line.